README.md

```diff
@@ -2,36 +2,92 @@
 
 CLI admin interface for [Cog](https://github.com/operable/cog).
 
-## Compiling
+## Setting up
 
-        mix escript
+### Prerequisites
+
+To compile and run cogctl you will need both Elixir and Erlang installed. If you're on osx the easiest way to install is through homebrew, `brew install elixir`. Elixir runs in the Erlang VM and therefore Erlang is required. If you're installing with homebrew, don't worry, it should pull Erlang as a dependency and install everything for you. Cogctl has been tested with Erlang version 18.3.2 and Elixir version 1.3.1.
+
+To build cogctl you'll also need hex and rebar:
+
+    mix local.hex --force
+    mix local.rebar --force
+
+### Compiling
+
+After getting all the prerequisites out of the way you can pull cogctl's deps and compile.
+
+    mix deps.get
+    mix deps.compile
+    mix escript
```

Comment:
I guess it might still be worth showing those other two commands, but it might not be necessary. Not a big deal either way.